src/index.ts

```diff
@@ -1,2 +1,29 @@
-export * from './function';
-export * from './api-gateway';
+import { ApiGatewayAuthorizer } from "./api-gateway";
+import { FunctionHandler } from "./functionHandler";
+import { Http } from "./http";
+import {
+    Budget,
+    CloudLogging,
+    ContainerRegistry,
+    DataStreams,
+    IotCore,
+    MessageQueue,
+    ObjectStorage,
+    Timer,
+} from "./triggers";
+
+export namespace Handler {
+    export type ApiGatewayAuthorizer = FunctionHandler<
+        ApiGatewayAuthorizer.Event,
+        ApiGatewayAuthorizer.Result
+    >;
+    export type Http = FunctionHandler<Http.Event, Http.Result>;
+    export type ObjectStorage = FunctionHandler<ObjectStorage.Event>;
```

Comment:
I believe we can use `Http.Result` for all trigger handlers.

Reply:
@esseswann this one is not resolved yet (

Reply:
I'm not sure that Http.Result is required for triggers, it does not break anything if something else is returned

Reply:
At the moment it doesn't, but this might change in future. Current convention is "return regular HttpResult from trigger function".